src/olympia/pages/templates/pages/shield_study_2.html

```diff
@@ -26,7 +29,7 @@
       <div class="details-header content-wrapper">
         <h1 data-hook="title">Help Make Firefox Faster</h1>
         <div class="experiment-controls">
-          <a class="button primary large" href="https://addons.mozilla.org/firefox/downloads/latest/x-screen-performance-study-1/">Start Performance Study</a>
+          <span id="opt-in" class="button primary large">Start Performance Study</span>
```

Comment:
This was an attempt to treat this like a button with no context menu.  Eventually we may be able to install from the Shield addon itself, so I wanted this to seem more "native" in feel.  Although now that I actually research it looks like the "correct" was to do this is setting oncontextmenu.  That said, I'm happy to go back to a hyperlink; it was a minor detail to me.